hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1767,6 +1767,48 @@ To disable checksum verification in `distcp`, use the `-skipcrccheck` option:
 hadoop distcp -update -skipcrccheck -numListstatusThreads 40 /user/alice/datasets s3a://alice-backup/datasets
 ```
 
+### <a name="distcp"></a> Using the S3 CRT client
+
+The [AWS CRT-based S3 client](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/crt-based-s3-client.html) 
+is built on top of the AWS Common Runtime (CRT), is an alternative S3 asynchronous client. It has
+enhanced connection pool management, and can provide higher transfer from and to S3 due to its 
```

Comment:
higher transfer what? bandwidth, latency, support calls?. 

also, s3a block output stream splits PUT requests, and vector io/analytics does the GET stuff, so what else does it offer? load balancing?